Packages/Status/Sources/Status/Row/StatusRowViewModel.swift

```diff
@@ -26,7 +29,9 @@ public class StatusRowViewModel: ObservableObject {
 
   @Published var translation: String?
   @Published var isLoadingTranslation: Bool = false
-
+  
+  @Published var seen = false
```

Comment:
It doens't need to be published since it have no impact on the UI.

Reply:
Yep.